snap-product-library-v2/src/test/java/org/esa/snap/product/library/v2/database/AllLocalFolderProductsRepositoryTest.java

```diff
@@ -37,11 +37,11 @@ public void testGetParameters() {
 
         parameter = findQueryParameterByName(AllLocalFolderProductsRepository.START_DATE_PARAMETER, queryParameters);
         assertNotNull(parameter);
-        assertEquals(Date.class, parameter.getType());
```

Comment:
The STTM annotation can be added to this test method.

Reply:
Ups! This was me. Was logged in with the senbox account.